deploy/testenv-deploy.js

```diff
@@ -0,0 +1,20 @@
+const env = require('./src/loadEnv')
+
+const deployToken = require('./src/utils/deployERC20Token')
+
+const mode = process.argv[2]
+
+async function main() {
+  switch(mode) {
+    case 'token':
+      const token = await deployToken()
```

Comment:
`token` variable is not used, so maybe we can avoid storing the result of the method:
```suggestion
      await deployToken()
```